docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md

```diff
@@ -26,7 +26,7 @@ under the License.
 
 # Streaming ELT from MySQL to Doris
 
-This tutorial is to show how to quickly build a Streaming ELT job from MySQL to Doris using Flink CDC, including the
+This tutorial is to show how to quickly build a Streaming ELT job from MySQL to StarRocks using Flink CDC, including the
```

Comment:
For what reason have modifications been made here?

Reply:
Good catch! I just copy all documentations from `docs/content/docs` to `docs/content.zh/docs`. We need to trace back to the original author

Reply:
+1

> For what reason have modifications been made here?

Reply:
> Good catch! I just copy all documentations from `docs/content/docs` to `docs/content.zh/docs`. We need to trace back to the original author

Looks like I made the typo, o(╯□╰)o

Reply:
Please help fix it，sir

Reply:
Fixed in the followup commit.